bin/gem2rpm

```diff
@@ -9,6 +9,11 @@ require 'fileutils'
 require 'open-uri'
 require 'uri'
 
+def show_err(message, options)
```

Comment:
This is good idea to extract this into separate method, but it is wrong place. The executable should be kept as slim as possible, so you should probably move it into lib/gem2rpm.rb.

BTW you probably want to have the $stderr replaced by some default parameter, defaulting to $stderr, that way, it should be possible write a test for this method (which I would appreciate BTW :))